modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradePortletDisplayCheck.java

```diff
@@ -0,0 +1,181 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradePortletDisplayCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String afterFormat(
+		String fileName, String absolutePath, String content,
+		String newContent) {
+
+		if (fileName.endsWith(".jsp")) {
+			newContent = addNewImportsJSP(
+				newContent,
+				"com.liferay.portal.configuration.module.configuration." +
+					"ConfigurationProviderUtil");
+		}
+		else {
+			newContent = addNewImports(newContent);
+		}
+
+		return newContent;
+	}
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		String newContent = content;
+
+		if (fileName.endsWith(".jsp")) {
+			return _formatJSP(content);
+		}
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			Matcher getPortletInstanceConfigurationMatcher =
+				_getPortletInstanceConfigurationPattern.matcher(
+					javaMethodContent);
```

Comment:
@TamyrisTorres, let's try to pursue consistency in this file:

- Why did you create a dedicated method to format JSP and not to format Java?
- Why did you name the `Matcher` here following a naming pattern different from the one chosen in the method `_formatJSP`?

Reply:
resolved